.github - Copy/workflows/linux.yml

```
@@ -0,0 +1,133 @@
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  build-gcc9:
+    runs-on: ubuntu-latest
+    name: GNU GCC 9
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)